src/site/content/en/progressive-web-apps/install-criteria/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Criteria for installability
+authors:
+  - petelepage
+date: 2020-02-14
+updated: 2020-02-14
+description: |
+  Progressive Web App installability criteria.
+draft: true
+---
+
+Many browsers indicate to the user that your Progressive Web App (PWA) is
+installable when it meets certain criteria. Example indicators include an
+install button in the address bar, or an Install menu item in the overflow
```

Comment:
"install button" is lowercase whereas "Install menu" is uppercase, seems unintentionally inconsistent

Reply:
```suggestion
Install button in the address bar, or an Install menu item in the overflow
```